KomodoPlatform/komodo-wallet-desktop
# fix toast imports

Fix for toast color bug reported in RC testing.

To test:
- place an order
- confirm the toast (middle bottom) displays with correct colors
